incubator/hnc/pkg/stats/metrics.go

```diff
@@ -2,22 +2,45 @@ package stats
 
 import (
 	"context"
+	"math"
+	"time"
 
 	ocstats "go.opencensus.io/stats"
 	ocview "go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
 var (
-	hierConfigReconcileTotal = ocstats.Int64("hierconfig_reconcile_total", "The number of HierConfig reconciliations happened", "times")
-	hierConfigWritesTotal    = ocstats.Int64("hierconfig_writes_total", "The number of HierConfig writes happened during HierConfig reconciliations", "times")
+	hierConfigReconcileTotal      = ocstats.Int64("hierconfig_reconcile_total", "The total number of HierConfig reconciliations happened", "reconciliations")
```

Comment:
Maybe worth some higher-level comment that each of these stats has exactly one view (if that's correct)